src/Analysis/Codelyzer.Analysis.Common/FileUtils.cs

```diff
@@ -25,11 +24,6 @@ public static string ReadFile(string pathFile)
         {
             return File.ReadAllText(pathFile);
         }
-
-        public static void CreateDirectory(string path)
```

Comment:
What is our process for changing public APIs? I see this in included with the nuget package published by Codelyzer (and consumed by CTA). I didn't see this method used in CTA, but that's not to say it isn't used somewhere. Should I instead mark it as deprecated and we can remove it in a major package version change?

Reply:
No strict process. Instead, any files with meaningful public APIs require approval from Code Owners. 

In this case, I don't believe CreateDirectory() is used in other repos so this should be a safe change. Worst case scenario it will be a trivial refactor.